apps/testingaccessibility/src/hooks/use-current-ability.ts

```diff
@@ -0,0 +1,14 @@
+import {useSession} from 'next-auth/react'
+import {getCurrentAbility} from '../server/ability'
+
+/**
+ * Uses next-auth session data to return a viewer's current ability.
+ */
+export function useCurrentAbility() {
+  const {data: sessionData, status} = useSession()
+
+  return {
+    status,
+    ability: getCurrentAbility({rules: sessionData?.rules}),
```

Comment:
most of the rules require that the user is authenticated, but there might be universal rules or defaults where an ability would only be true for an unauthenticated user, for instance

currently all of our abilities are related to purchases and auth'd users

Reply:
ConvertKit stuff would fit great in here as well? Pre-launch abilities are all CK driven.